### PR TITLE
Support for <base>/game for GOG Linux native games

### PR DESCRIPTION
**Pull request by sluedecke**

### CODE REVIEW

**Comment by mtkennerly** [comment]
Thanks!

Following up from https://github.com/mtkennerly/ludusavi-manifest/issues/11:

> Both Gog and Gog Linux are fine IMO, but the later one makes it clearer to me. Else one could rename Gog to "Gog Offline" since it is an offline installer I get on Linux. But it should be checked if the offline installers on Windows exhibit the same behavior.

The GOG option is already meant to be for the standalone/offline installers, so that's why I'd prefer not to add another variant. We could rename it, although for now I'd rather just clarify it in the documentation if necessary.

I just tested a couple of offline installers on Windows (Dead Cells, Tyrian 2000), and neither one includes the `game` subfolder, so I think we can assume Linux only for now.

File: src/prelude.rs [hunk 1]
```diff
@@ -402,6 +402,12 @@ pub fn parse_paths(
             .replace("<regHkcu>", SKIP)
             .replace("<regHklm>", SKIP),
     );
+    if root.store == Store::GogLinux {
```
Comment (mtkennerly) [inline src/prelude.rs:405]: Let's change this to `if root.store == Store::Gog && get_os() == Os::Linux` and remove the `GogLinux` variant.

File: src/config.rs [hunk 2]
```diff
@@ -388,6 +388,8 @@ impl Config {
             // GOG Galaxy:
             (format!("{}/GOG Galaxy/Games", pf32), Store::GogGalaxy),
             (format!("{}/GOG Galaxy/Games", pf64), Store::GogGalaxy),
+            // GOG Linux:
+            ("~/GOG Games".to_string(), Store::GogLinux),
```
Comment (mtkennerly) [inline src/config.rs:392]: Nice :+1: Always good to have more of these known locations.

**Review by mtkennerly — CHANGES_REQUESTED** [review]

File: src/config.rs (hunk 2 above)
Comment (sluedecke) [reply src/config.rs]: Dropped since already included in Store::Gog

**Comment by sluedecke** [comment]
Hm ... I am new to the pull request process on github, pushing the changes as requested should be sufficient, right?

**Comment by mtkennerly** [comment]
> Hm ... I am new to the pull request process on github, pushing the changes as requested should be sufficient, right?

Yep! Everything looks great. Thanks again for your help \:D

**Pull request merged**